Resources/Textures/Clothing/Head/Hardsuits/goliathhelm.rsi/meta.json

```diff
@@ -2,7 +2,6 @@
     "version": 1,
     "license": "CC-BY-SA-3.0",
     "copyright": "Normal and vox sprites created by https://github.com/Pronana / princesscheeseballs (Discord). Dog states modified from base states by Sparlight (GitHub).",
-
```

Comment:
It's good that you are removing unnecessary linebreaks, but it's out of scope in this PR. It will also merge-conflict with the other PR when you add the new attribution. Move this fix to the other PR, and I'll merge both PRs